packages/target-tools/src/perftool.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/prefer-default-export */
-import now from "performance-now";
 import { isDefined, isUndefined } from "./utils";
+import now from "./now";
```

Comment:
That's fine.  I can still use this package for node.js.  But it won't work in the browser, so it doesn't make sense to package it into at.js as part of target-tools.  Do you know if there's a way to exclude it from the at.js package?  I'll have to check.

Reply:
any code not used in at.js is tree-shacken out.  So you don't have to worry too much.  It's why `target-tools` exports things the way it does -- so that if something isn't imported it won't be included in the build.

However whether you use `performance-now` or your own `now` implementation, it will be included in at.js because decisioning engine uses it.  However, the code is written to just fall back to performance.now if it exists in browser (which it does).  bundle size is negligible in this instance.  But if you really want to focus on bundle size you can pass in a `performance` object the same way we do with `fetch`.  That way at.js passes in native performance, and node passes in the performance-now object.  but i think it is probably only 1K or less savings.

Heres the implementation code for `performance-now`  not much there and you can see it uses native performance object if it is available...

<img width="837" alt="screenshot-2021-10-11-163137@2x" src="https://user-images.githubusercontent.com/427213/136863325-17859c90-4959-4c22-afa1-d91c595ab531.png">